app/assets/stylesheets/views/_service-manual-guide.scss

```diff
@@ -0,0 +1,145 @@
+// Proposed component:'govuk_component/breadcrumb'
+
+.govuk-breadcrumb {
+  ol {
+    @extend %contain-floats;
+    padding-top: 0.75em;
+    padding-bottom: 0.75em;
+    li {
+      @include core-16;
+      float: left;
+      background-image: file-url("separator.png");
+      @include device-pixel-ratio() {
+        background-image: file-url("separator-2x.png");
+        background-size: 6px 11px;
+      }
+      background-position: 100% 50%;
+      background-repeat: no-repeat;
+      list-style: none;
+      margin-right: 0.5em;
+      margin-bottom: 0.4em;
+      margin-left: 0;
+      padding-right: 1em;
+      a {
+        color: $text-colour;
+      }
+      strong {
+        font-weight: normal;
+      }
+      &:last-child {
+        background-image: none;
+        margin-right: 0
+      }
+    }
+  }
+}
+
+
+// Modify existing component: 'govuk_component/title'
+
+.govuk-title {
+  margin-top: 1.875em; // 30/16
+  margin-bottom: 1.875em; // 30/16
+  @include media(tablet) {
+    margin-bottom: 2.5em; // 40/16
+  }
+  .govuk-title-context {
+    @include core-24;
+    color: $secondary-text-colour;
+  }
+}
+
+.govuk-community-contact {
+  margin-top: 1.875em; // 30/16
+}
+
+
+// Modify existing component: 'govuk_component/metadata'
+
+.govuk-metadata {
+  margin-bottom: 2em;
```

Comment:
This is going to leak into Case Study pages, affecting their components.

| before | after |
| --- | --- |
|  ![screenshot 2015-10-28 11 16 31](https://cloud.githubusercontent.com/assets/63201/10786962/6a3a7d28-7d65-11e5-9adc-cd5cf9c4cf5b.png)  | ![screenshot 2015-10-28 11 16 27](https://cloud.githubusercontent.com/assets/63201/10786961/6a23b462-7d65-11e5-8c0e-91196bca2800.png) |

They'll potentially be other components this leaks into, eg, margins for `govuk-title`